code/modules/clothing/suits/f13armor.dm

```diff
@@ -1359,8 +1359,35 @@
 	body_parts_covered = CHEST|GROIN|LEGS|ARMS|HANDS
 	slowdown = 0.05
 
+//Fluff Stuff
 /obj/item/clothing/suit/armor/f13/rangercombat/finlayranger
```

Comment:
All of these are children of rangercombat in f13factionarmor.dm, meaning all of these are going to get veteran ranger armor essentially.